### PR TITLE
Add task for kotlin script files

### LINKED ISSUE #98: Add a task to check project kotlin dsl files
Currently, while `KtlintCheck` task supports checking `kts` files, plugin itself doesn't add any task for this files.

Plugin should add task that checks project kotlin gradle dsl files.

### PR DESCRIPTION
**Pull request by Tapchicoma**
Fixes #98 

@JLLeitschuh can you check if I am doing right with extension and additional paths for kotlin script files?

### CODE REVIEW

File: plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintBasePlugin.kt [hunk 1]
```diff
@@ -24,11 +27,27 @@ open class KtlintBasePlugin : Plugin<Project> {
             }
         }
 
+        val kotlinScriptAdditionalPathApplier = { fileCollection: FileCollection ->
```
Comment (JLLeitschuh) [inline plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintBasePlugin.kt]: Do you want to be explicit about the type here, using `KotlinScriptAdditionalPathApplier` just for sanities sake? If you don't think it adds any value, feel free to not add it.

File: plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintBasePlugin.kt [hunk 2]
```diff
@@ -24,11 +27,27 @@ open class KtlintBasePlugin : Plugin<Project> {
...
+            val configureAction = Action<Task> {
+                with(it as KtlintCheckTask) {
+                    fileCollection.forEach {
+                        val additionalPath = project.fileTree(it)
+                        additionalPath.include("*.kts")
+                        source = source.plus(additionalPath)
+                    }
```
Comment (JLLeitschuh) [inline plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintBasePlugin.kt]: Do you really need to iterate over the entire `fileCollection` here? I feel like that will just slow down the configuration phase. If there's some way to get away without iterating over the entire thing, I think that would be optimal.

Comment (Tapchicoma) [reply plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintBasePlugin.kt]: Changed to `List<String>` in extension, so applier creates a `FileTree` from provided paths.

**Comment by Tapchicoma** [comment]
@JLLeitschuh can you recheck this PR again?

**Comment by Tapchicoma** [comment]
Yes, I will make a new release and then apply it on plugin project.

**Pull request merged**